src/pages/commits/ActiveCommits/CommitChart.tsx

```diff
@@ -57,3 +95,46 @@ const Bar = styled.div<BarProps>`
   background-color: ${({ color }) => color};
   width: 13px;
 `;
+
+const TooltipContainer = styled.div`
+  width: 150px;
+  height: 160px;
+  margin: auto;
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  flex-direction: column;
+  background-color: ${gray.light3};
+`;
+
+const Number = styled(Disclaimer)`
+  width: 40px;
+  font-weight: bold;
+  font-size: 9px;
+  letter-spacing: 0.15px;
+  text-align: center;
+  position: absolute;
+  margin-left: 120px;
```

Comment:
[nit] It's preferable to avoid absolute spacing in a situation like this where we really want to position the elements relative to each other. One approach here could be to create a container like `ColorRow` and then use flexbox and its [`space-between`](https://css-tricks.com/snippets/css/a-guide-to-flexbox/#justify-content) property to justify the circle, label, and count.